client/src/common/date.ts

```diff
@@ -3,19 +3,20 @@
 import { DateTime } from 'luxon';
 import { ComposerTranslation } from 'vue-i18n';
 
-export function formatTimeSince(date: DateTime | undefined, t: ComposerTranslation, d: any, defaultValue='', format='long'): string {
+export function formatTimeSince(date: DateTime | undefined, t: ComposerTranslation, d: any, defaultValue='', format:'long'|'short' = 'long',
```

Comment:
Some inconsistencies in spaces that the linter apparently doesn't catch. The right format is `var: type = value` with those precise spaces.